tests/.gitignore

```diff
@@ -0,0 +1,52 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
```

Comment:
Why do we need `.gitignore` file in `tests/` directory?
The planner root directory already has a gitignore file

Reply:
Removed.